apps/site/pages/docs/getting-started/0-getting-started-overview.mdx

```diff
@@ -7,57 +7,60 @@ import { Callout } from 'nextra-theme-docs'
 
+[Open a ticket with the VTEX support team](https://help.vtex.com/support) and request the installation of the FastStore Onboarding app.
 
-**3. The FastStore Onboarding app must be installed in your store account.** 
+---
 
-The app is where you start your FastStore project, and to request the installation of the app, [open a ticket with the VTEX support team](https://help.vtex.com/support).
+## Track overview
 
----
+In this section, you will find a concise summary of the FastStore Getting Started track. It provides an outline of the topics covered in each step, giving you a clear roadmap for your journey.
   
-## 1. [FastStore Onboarding](/docs/getting-started/1-faststore-onboarding/overview)
+### 1. [FastStore Onboarding app](/docs/getting-started/1-faststore-onboarding/overview)
 
-Start your project with the FastStore Onboarding app. It streamlines the process by setting up the essential store configurations, creating a Github repository for your store, and providing a live demo preview of your store.
+Create your project with the FastStore Onboarding app. This app streamlines the creation of a FastStore project by configuring essential store settings, creating a dedicated GitHub repository for your project, and defining a default domain for your store.
 
-## 2. [Setting Up the Project](/docs/getting-started/2-setting-up-the-project)
+### 2. [Setting Up the Project](/docs/getting-started/2-setting-up-the-project)
 
-Discover the core technologies that we will be using in this tutorial and set up all the necessary tools to implement your very first FastStore storefront and create your first storefront customization.
+Discover the core technologies used in this tutorial and set up all the necessary tools to create your very first FastStore storefront.
 
-## 3. [Understanding the Project Structure](/docs/getting-started/3-project-structure)
+### 3. [Understanding the Project Structure](/docs/getting-started/3-project-structure)
 
-Find out how to organize your FastStore project and get to know our recommended conventions for naming files and folders.
+Learn how to organize your FastStore project and get to know our recommended naming conventions for files and folders.
 
-## 4. [FastStore CLI](/docs/getting-started/4-faststore-cli)
-The FastStore Command Line Interface (CLI) is the main pre-configured tool for starting, building, and developing FastStore projects.
+### 4. [FastStore CLI](/docs/getting-started/4-faststore-cli)
+Install the FastStore Command Line Interface (CLI), a development tool for starting, building, and developing FastStore projects.
 
-## 5. [Next Steps](/docs/getting-started/5-next-steps)
+### 5. [Next Steps](/docs/getting-started/5-next-steps)
   
-Next, check out the next section for further guidance on how to enhance your store.
+Explore additional resources and recommendations to enhance various aspects of your store, including performance and overall user experience.
```

Comment:
Achei essa descrição um pouco genérica, deixei essa sugestão para ser mais específico sobre o que tem na seção.
Coloquei performance e UX, mas pode alterar de acordo com o que for mais adequado